### DISCUSSION TITLE
Email OTP

### DISCUSSION THREAD

**Discussion by shivani1230** [cypress-io/cypress#28626] [Plugins]
We are implementing cypress on our existing project where we have JAVA as a backend tech from where email OTP is triggered and validated and frontend is Angular based. In such case where we want to validate email via cypress what option do we have? Through cypress we are redirecting to signup page then it enters email and password field on click of submit button OTP is send on specified email, how do we get that OTP and validate through cypress? Or is there a way where we can set a generalized OTP and that can bypass the functionality without any obstacle?

**Comment by dsai05q** [comment]
did you find any way to otp extraction from email? [shivani1230](https://github.com/shivani1230)
